activerecord/test/cases/adapters/mysql2/active_schema_test.rb

```diff
@@ -180,6 +180,19 @@ def test_indexes_in_create
     assert_match expected, actual
   end
 
+  def test_forceful_temporary_table_creation
+    expected_sql = [
+      /\ADROP TEMPORARY TABLE IF EXISTS `forced_temp`/,
+      /\ACREATE TEMPORARY TABLE `forced_temp` \(`id` bigint NOT NULL AUTO_INCREMENT PRIMARY KEY\)/
+    ]
+
+    assert_sql(*expected_sql) do
+      ActiveRecord::Base.connection.create_table(:forced_temp, temporary: true, force: true)
+    end
+  ensure
+    ActiveRecord::Base.connection.drop_table :forced_temp, temporary: true rescue nil
```

Comment:
I think you can use `if_exists: true` to ensure it is dropped only when present instead of `rescue nil` which can swallow other exceptions. Or is there any other reason to use `rescue nil` here?

Reply:
I followed the approach used elsewhere in this file for consistency, but I think you're right.

```ruby
ActiveRecord::Base.connection.drop_table :delete_me rescue nil
```